proxy/ControlBase.cc

```diff
@@ -411,7 +410,7 @@ struct TextMod : public ControlBase::Modifier {
 TextMod::TextMod() : text() {}
 TextMod::~TextMod()
 {
-  free(text.data());
```

Comment:
I dislike `const_cast`. In this case would a `char const*` work as well? Another alternative would be `MemSpan<char>` which provides a `view` method to extract a view when needed.